packages/react-variant-view/src/util/MutationDataUtils.ts

```diff
@@ -0,0 +1,140 @@
+// functions in this component come from signal
```

Comment:
We should probably rename this to `SignalMutationUtils` and move into `cbioportal-utils`